Add sample images

ReedyBear opened the pull request:
It seems like this project is still young, so I don't know if you actually want this yet, but I was poking around to see what output it produced, and thought I might as well send a PR while I'm at it. Also wasn't sure if you'd want "docs" folder or "examples" folder or what, so just made a guess.

ReedyBear committed:
Add sample images

ReedyBear committed:
Added example images to README, moved them into docs folder

P5vc commented:
Yes, that is true: this repository is still quite young, and it may be awhile before we implement it into our website. However, we are still glad that you checked it out and contributed, so thank you!

I definitely agree with you that we should have sample images included within the `README`, however if you don't mind, I would change two things:

1. The images should be in `PNG` format instead of `JPEG`, as `PNG` is lossless (and therefore won't have any compression artifacts in the image), unlike `JPEG`.
1. I think that `examples` may be a better name for the folder, as the only `docs` I foresee us having is the main `README`.

If you would like to generate these new images and update the pull request, great! If not, just let me know and I have no problem doing it. I would also appreciate it if you could add the typo corrections to the pull request; I saw you committed them to your fork of `BotBlock`, and that was quite helpful!

Taeluf commented:
Oh cool! I'll modify it! Probably tomorrow afternoon. 

Lolyeah i added those then ammended a commit message, which added two extra commits, and i didn't want to learn how to delete old commits or send ugly commits, so i just rolled it back and by then i didn't wanna mess with it any more lol.

I'll add the typo fixes back in too! ☺

Taeluf commented:
I'm not sure if PRs can be modified, but I'm thinking we close this one & i just create a brand new one? Not entirely sure how all that works.

ReedyBear committed:
made pngs and examples dir

ReedyBear committed:
minor typo/grammar fix

P5vc commented:
> Oh cool! I'll modify it! Probably tomorrow afternoon.

Great! No rush!

> Lolyeah i added those then ammended a commit message, which added two extra commits, and i didn't want to learn how to delete old commits or send ugly commits, so i just rolled it back and by then i didn't wanna mess with it any more lol.
> 
> I'll add the typo fixes back in too! ☺

No worries! If you want to remove the repository and then fork it again, that's totally fine. You're more than welcome to open a new pull request if that's easier

P5vc commented:
> 
> 
> I'm not sure if PRs can be modified, but I'm thinking we close this one & i just create a brand new one? Not entirely sure how all that works.

Any commits you make to the `add_sample_images` branch you created will automatically be reflected in this pull request. So, feel free to make any changes you'd like to the `add_sample_images` branch, and you won't need to open a new pull request. When this pull request gets merged, it will appear as only a single commit, so you won't have to worry about "messy commits". However, if you'd rather "start over" and open a new pull request, that's fine too.

P5vc commented:
Wow, we both made changes at about the same time! :) I'll take a look at those new commits now. 👍

Taeluf commented:
Oh, nice! The png example? I considered adding something about that

P5vc commented on README.md:
````diff
@@ -65,3 +65,8 @@ BotBlock.hashText = True
 BotBlock.generate('/tmp/')
 ```
+
+### Example Images
````
```suggestion
### Example Images

```

As part of our coding standards, we always space-out blocks in markdown

P5vc commented on README.md:
````diff
@@ -65,3 +65,8 @@ BotBlock.hashText = True
 ```
+
+### Example Images
+![Sample Captcha #1](/examples/sample1.png)
````
```suggestion
![Sample Captcha #1](/examples/sample1.png "First Sample Image")
```

Add default text for all of the images, in case they don't load

P5vc reviewed (changes requested):
Looks good overall! The only thing I would change (other than the in-line suggestions I made) would be to re-generate the first image, as part of the last character got cutoff

ReedyBear committed:
Added titles to sample captcha images

Didn't even know title was an img attribute. https://blog.spotibo.com/difference-between-alt-text-and-title-text/ had some good looking information on this topic

ReedyBear committed:
spaced out code block

Co-authored-by: Priveasy <Admin@Priveasy.org>

P5vc commented:
@Taeluf Alright, this looks good, thanks again! I'm going to go ahead and merge this pull request.

If you're interested, let us know, and you can claim an email and/or premium Priveasy Docs account on our Supporters Page (https://github.com/P5vc/Documentation/blob/master/About/Supporters.md) for 30 days. Feel free to keep contributing across our repositories if it's something you enjoy, and we can extend that time and offer you even greater perks! :)
